codespell_lib/data/dictionary.txt

```diff
@@ -34732,6 +34732,7 @@ mezmorizing->mesmerizing
 miagic->magic
 miagical->magical
 mial->mail
+mian->main
```

Comment:
Perhaps `moan` as well?
```suggestion
mian->main, moan,
```

Reply:
Compared to "main", "moan" doesn't seem to be a commonly used programming word, but it seems to work.
Thanks!

Reply:
codespell is not used exclusively for code. Not sure what you mean by "it seems to work".

Reply:
Adding "moan" is a good suggestion, thanks!

Reply:
Then could you apply the change?